great_expectations/core/expectation_configuration.py

```diff
@@ -1035,36 +1039,30 @@ def patch(self, op: str, path: str, value: Any) -> "ExpectationConfiguration":
         return self
 
     @property
-    def ge_cloud_id(self):
-        if hasattr(self, "_ge_cloud_id"):
-            return self._ge_cloud_id
-        else:
-            return None
+    def ge_cloud_id(self) -> Optional[str]:
+        return self._ge_cloud_id
```

Comment:
Since this change was to prevent the attr from showing up when serialized (when `None`), I've removed this.

The same functionality is available through `@post-dump`